lib/c-api/tests/wasmer-c-api-test-runner/src/lib.rs

```diff
@@ -198,7 +198,7 @@ fn test_ok() {
             // compile
             let output = command
                 .output()
-                .expect(&format!("failed to compile {command:#?}"));
+                .unwrap_or_else(|_| panic!("failed to compile {command:#?}"));
```

Comment:
Use expect instead

Reply:
this is a clippy autofix done by clippy --fix --all, it does exactly the same thing, but it doesn't construct a string in the "okay" case. wontfix because it will be reapplied by `clippy --fix` on the next run anyways.